Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests
-RUN curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current
+RUN curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current-tripleo
```

Comment:
doesn't match the commit message?

Reply:
matches commit message, but not PR summary, i guess.  Which done did you intend?

Reply:
Ahh, the summary was created for the first version of the patch, turns out "tripleo_repos" doesn't take "current-tripleo-rdo" as a param. So I switched it to current-tripleo, which wasn't my original intention but still safer then what we currently use.